### PR TITLE
Fix regressions in discussions

### PR DESCRIPTION
**Pull request by francisschmaltz**
Fixes: #1849 
Fixes: #1847 
Fixes: #1846 

Also fixes odd light theme UI issues.

screen shot 2019-01-13 at 12 28 27 pm
screen shot 2019-01-13 at 12 28 16 pm

### CODE REVIEW

File: web/src/discussions/DiscussionsList.tsx [hunk 1]
```diff
@@ -23,16 +24,20 @@ const DiscussionNode: React.FunctionComponent<DiscussionNodeProps> = ({ node, lo
     const openDiscussion = (e: any) => openFromJS(inlineURL, e)
     const preventDefault = (e: React.MouseEvent<HTMLAnchorElement>) => e.preventDefault()
     const stopPropagation = (e: React.MouseEvent<HTMLAnchorElement>) => e.stopPropagation()
+
+    console.log(withRepo)
```
Comment (slimsag) [inline web/src/discussions/DiscussionsList.tsx]: remove

Comment (francisschmaltz) [reply web/src/discussions/DiscussionsList.tsx]: 👍

File: web/src/discussions/DiscussionsList.tsx [hunk 2]
```diff
@@ -23,16 +24,19 @@ const DiscussionNode: React.FunctionComponent<DiscussionNodeProps> = ({ node, lo
...
-                <a href="#" onClick={preventDefault} className="discussions-list__row-title ">
-                    {node.title}
-                </a>
-                <small className="discussions-list__row-id">#{node.id}</small>
+                <h3 className="discussions-list__row-title ">
+                    <a href="#" onClick={preventDefault}>
+                        {node.title}
+                    </a>
+                    <small className="discussions-list__row-id">#{node.id}</small>
+                </h3>
```
Comment (slimsag) [inline web/src/discussions/DiscussionsList.tsx]: What is the motivation behind this change? I think if you click the ID, it should bring you to the actual item. That better drives home that the ID is the item, I think

Comment (slimsag) [reply web/src/discussions/DiscussionsList.tsx]: Ah, hmm, I see you didn't change that and GitHub does the same, nevermind.
